others/Longest_Substring_Without_Repeating_Characters.cpp

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Please follow https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md
for the proper documentation style

Reply:
Is there any other changes I need to make as I have updated the code added testcases and document the part.